en/docs/setup/deployment-patterns.md

```diff
@@ -1,20 +1,23 @@
 # Deployment Patterns
 
 The following sections provide high level information on
-the recommended deployment pattern. 
+the recommended deployment patterns. 
```

Comment:
Since we have only one deployment pattern, lets keep it singular

Reply:
and shall we also rename this in the mkdocs.yaml to "Overview" instead of "Deployment patterns"

Reply:
Fixed: c0596ca7bc5ef3dad2c46529bde1c9eba89141f5